src/commands/deploy/deploy.ts

```diff
@@ -536,7 +536,7 @@ const runDeploy = async ({
 
   if (!deployToProduction) {
     functionLogsUrl += `?scope=deploy:${deployId}`
-    edgeFunctionLogsUrl += `?scope=deploy:${deployId}`
+    edgeFunctionLogsUrl += `?scope=deployid:${deployId}`
```

Comment:
Maybe we do a follow-up PR to make these two match? The `deploy` naming seems better.

Reply:
The downside is we'd effectively have to support both keys forever. Well, maybe not forever, but we'd have to do a two-phase change with several months in between.